### ISSUE TITLE
Remember Runner settings

### ISSUE THREAD

**Issue by d-schmidt** [a85/POSTMan-Chrome-Extension#9558]
**Is your feature request related to a problem? Please describe.**
Every single time I have to click all 4 checkboxes in the runner. Can this tool please remember my choice?

**Describe the solution you'd like**
Next runner has the same settings as the last.

**Describe alternatives you've considered**
There are none, this is such basic feature.

**Additional context**
I always use these, default is inverse:
![grafik](https://user-images.githubusercontent.com/3226483/107034313-d3cbf500-67b6-11eb-8153-48d2c1929293.png)

**Comment by DannyDainton** [comment]
Hey @d-schmidt 👋 

Each time a new Runner tab is opened, it will revert back to the default setting state - If you have just run a Collection and you hit the `New` button in the Runner tab, it will store the same settings. 

If you take a look in the `History` you will be about to see the previous Collection Runs, these will also have the setting information set for that run when you hit the `New` button in the Runner tab.

I believe this issue is more around controlling the default Runner settings, rather than the Runner never remembering the settings :D

**Comment by d-schmidt** [comment]
Yeah I'm also wondering why these are not the default settings. Having the Runner not interact with my workspace seems way better. But maybe it is because i'm mainly using it for integration tests.

**Comment by d-schmidt** [comment]
Once again I tried to punch postman for reverting back to the subjectively false default settings every single time I run a collection. Noo I don't want to keep the test cookies. No I don't want to keep variables from tests. Please, fix this. No I don't want it to use existing environment cookies. These are tests, they need to run deterministic each time.
